include/yarpWholeBodyInterface/yarpWholeBodyInterface.h

```diff
@@ -35,7 +35,7 @@ namespace wbi {
 
 namespace yarpWbi {
     class yarpWholeBodyActuators;
-    class yarpWholeBodyModel;
+    class yarpWholeBodyModelV1;
```

Comment:
This have been fixed in https://github.com/robotology/yarp-wholebodyinterface/pull/59/commits/ef575401844085eba01500e14da36031f509fff6 , except for that forward declaration that is superfluous given the includes provided (see https://github.com/robotology/yarp-wholebodyinterface/commit/aae63c9b0a950113a0d9f3d431ebe67a0f8ed171). Removing all the forward declaration should solve the issue.